crates/services/p2p/src/service.rs

```diff
@@ -260,6 +278,13 @@ impl SharedState {
         Ok(())
     }
 
+    pub fn update_block_height(&self, block_height: BlockHeight) -> anyhow::Result<()> {
```

Comment:
We'll need to update the p2p adapter during the integration with importer to have it update the block height. Alternately, we could just add a db port for latest block height in p2p and fetch it on-demand instead of notifying the p2p service of block height changes.

Reply:
Hmm, previously, we discussed that the block importer notifies all listeners, and p2p is one of the listeners. It means that we need to pass a `BoxStream`=)

With latest height we will notify neighbors as soon as possible

Reply:
Do you mean the fuel sync not the importer? This should be a stream that is pushed to by p2p and is pulled from the sync

Reply:
yeah you're right @xgreenx this was just an initial iteration, I'll refactor it to BoxStream :)

Reply:
> Do you mean the fuel sync not the importer? This should be a stream that is pushed to by p2p and is pulled from the sync

I meant block importer=) This function is used to notify P2P about new heights, not the `fuel-core-sync`. For `fuel-core-sync` we have `subscribe_block_height` below=)